labs/item/internal/_item.scss

```diff
@@ -85,7 +85,7 @@
   }
 
   .default-slot,
-  ::slotted(*) {
+  :not([name='start'])::slotted(*) {
```

Comment:
Can we select all text slots instead? That makes more sense for what is expected

`.text ::slotted(*)`

Reply:
oof I missed that...